action.yml

```diff
@@ -0,0 +1,55 @@
+inputs:
+  targets:
+    description: 'depviz target (format: \"org1/repo1 org1/repo2 org2 repo1\")' 
+    required: true
+  branch: 
+    description: 'the branch where to push the generated files' 
+    default: 'depviz-generate'
+  gen-json: 
+    description: 'generate json output'
+    default: 'false'
+  gen-png: 
+    description: 'generate png output'
+    default: 'false'
+  gen-csv:
+    description: 'generate csv output'
+    default: 'false'
+
+
+runs:
+  using: 'composite'
+  steps:
+  - uses: actions/checkout@v3
+
+  - run: mkdir db
+    shell: bash
+
+  - uses: actions/setup-go@v4
+
+  - run: mkdir depviz-outputs
+    shell: bash
+  - run: go install moul.io/depviz/v3/cmd/depviz@latest
+    shell: bash
+  - run: depviz fetch --github-token ${GITHUB_TOKEN} ${{ inputs.targets }} 
```

Comment:
This part should run only once, even if we want to generate multiple outputs.

Reply:
Also, the generated `indexes.bolt` file should be cached between two runs.

Reply:
I think the format of the custom actions will require 2 actions:
```
- uses: depviz-action-fetch
  [...]
  - target: "depviz/depviz depviz/depviz-test"

- uses: depviz-action-gen
  - target: "depviz/depviz"

-  uses: depviz-action-gen
  - target: "depviz/depviz depviz/depviz-test"
```

Reply:
done :heavy_check_mark: